### DISCUSSION TITLE
Documentation not up to date?

### DISCUSSION THREAD

**Discussion by jmartens** [GitTools/GitVersion#4327] [Q&A] [answered]
You documentation website seems to mention v5.12.0 in its header and judging from https://github.com/GitTools/GitVersion/discussions/4326 and the 6.x release notes breaking changes section it is not up-to-date. Am I correct?

> The Mainline mode and the related implementation has been removed completely. The new Mainline version strategy should be used instead.

**Comment by arturcic** [comment, marked as answer]
We try to keep it up to date, but I can agree that we need to review the latest version of the state of the codebase
